### ISSUE TITLE
batch question about horovod

### ISSUE THREAD

**Issue by 652994331** [NVIDIA/DeepLearningExamples#368]
so Horovod can help us to increate the batch size to batch size each * num_workers. from the  code run_pretraining.py, we have to set the report loss = True in order to use this glocal_batch_size.Am i right guys? thanks for your reply, really appreciate.

**Comment by swethmandava** [comment]
global_batch_size = batch_size * num_workers irrespective of whether report_loss=True or False.

It is just printed to the log if report_loss=True and therefore defined only in that case.

**Comment by 652994331** [comment]
When i looked the code. if do_train part, the train_input_fn has batch_size=Flags.train_batch size, not global_batch_size. So i was wondering hwo can we know that we are using global_batch_size

**Comment by swethmandava** [comment]
dataset is sharded based on num_workers

https://github.com/NVIDIA/DeepLearningExamples/blob/aa061052c674ce19db4966854db196c209fa82e0/TensorFlow/LanguageModeling/BERT/run_pretraining.py#L445
